vispy/visuals/tests/test_rectangle.py

```diff
@@ -15,113 +15,123 @@
 def test_rectangle_draw():
     """Test drawing rectpolygons without transform using RectPolygonVisual"""
     with TestingCanvas() as c:
-        rectpolygon = visuals.Rectangle(pos=(50, 50, 0), height=40.,
-                                        width=80., color='red')
-        c.draw_visual(rectpolygon)
-        assert_image_approved("screenshot", 'visuals/rectpolygon1.png')
-
-        rectpolygon = visuals.Rectangle(pos=(50, 50, 0), height=40.,
-                                        width=80., radius=10., color='red')
-        c.draw_visual(rectpolygon)
-        assert_image_approved("screenshot", 'visuals/rectpolygon2.png')
-
-        rectpolygon = visuals.Rectangle(pos=(50, 50, 0), height=40.,
-                                        width=80., radius=10., color='red',
-                                        border_color=(0, 1, 1, 1))
-        c.draw_visual(rectpolygon)
-        assert_image_approved("screenshot", 'visuals/rectpolygon3.png')
-
-        rectpolygon = visuals.Rectangle(pos=(50, 50, 0), height=40.,
-                                        width=80., radius=10.,
-                                        border_color='white')
-        c.draw_visual(rectpolygon)
-        assert_image_approved("screenshot", 'visuals/rectpolygon4.png',
+        visuals.Rectangle(center=(50, 50, 0), height=40.,
+                          width=80., color='red',
+                          parent=c.scene)
+
+        assert_image_approved(c.render(), 'visuals/rectpolygon1.png')
+
+        visuals.Rectangle(center=(50, 50, 0), height=40.,
```

Comment:
The rectangles from each test are being displayed together. You need to either remove each rectangle before testing the next, or test a single rectangle and update its properties for each test.